docs/user-guide/layer-settings.md

```diff
@@ -134,6 +139,110 @@ The editor is easy to approach thanks also to the following functions:
 !!!warning
     The *autocomplete* and the *color picker* functions are available only in the CSS editor.
 
+### Visual Editor Style
+
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) also allows to edit the layers style using a *Visual editor* with a most user friendly UI.Clicking on the <img src="../img/button/visual_editor_style_button.jpg" class="ms-docbutton"/> button a section opens so that the user can customize the style through with a visual stile editor by adding editing symbolizers, which can be: *Mark*, *Icon*, *Line*, *Fill* and *Text*. It is anyway possible to switch to the text editor mode if necessary for a more complex styling.
+
+<img src="../img/layer-settings/visual_editor_style.jpg" class="ms-docimage"  style="max-width:500px;">
+
+Once a symbolizer has been added and customized, you can:
+
+<img src="../img/layer-settings/style_options.jpg" class="ms-docimage"  style="max-width:500px;">
+
+* **Filter** he style rule, as explained [here](filtering-layers.md#attribute-filter), by clicking the <img src="../img/button/filter_white_button.jpg" class="ms-docbutton"/> button.
```

Comment:
* **Filter** the style rule, as explained [here](filtering-layers.md#attribute-filter) in order to apply the style only to certain layer features. It is possible by clicking on the <img src="../img/button/filter_white_button.jpg" class="ms-docbutton"/> button.